### ISSUE TITLE
Proposal ordering should consider status

### ISSUE THREAD

**Issue by samuveth** [snapshot-labs/snapshot-hub#259]
Now proposals are purely ordered by creation date, but we should also consider the status and show active proposals first.

**Comment by chef-jojo** [comment]
It would be nice if we can order proposals by votes count

**Comment by mktcode** [comment]
@chef-jojo you mean like maybe an option in the UI to change the ordering? Maybe in general the find-ability of proposals could be improved, with a search input let's say.

**Comment by chef-jojo** [comment]
> @chef-jojo you mean like maybe an option in the UI to change the ordering? Maybe in general the find-ability of proposals could be improved, with a search input let's say.

I'm looking for graphql query Proposals `orderBy` here. Sorry if this is not a right place to ask

**Comment by mktcode** [comment]
@chef-jojo Oh.... we're in the hub repo... not the frontend. 😄 No, no... perfect place to ask.

**Comment by samuveth** [comment]
@bonustrack I was referring to this issue yesterday.

**Comment by zzuziak** [comment]
Hey @samuveth is this issue still applicable?

**Comment by samuveth** [comment]
@zzuziak yes, this is specially annoying when a space has a proposal that goes on for longer, as they will lose visibility over time.
